pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -1,23 +1,22 @@
-{ lib, stdenv, fetchFromGitHub, cmake, python3, vulkan-headers, pkg-config
-, xlibsWrapper, libxcb, libXrandr, libXext, wayland, addOpenGLRunpath }:
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, libX11, libxcb
+, libXrandr, wayland, vulkan-headers, addOpenGLRunpath }:
 
 stdenv.mkDerivation rec {
   pname = "vulkan-loader";
-  version = "1.2.162.0";
+  version = "1.2.176.0";
 
   src = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "Vulkan-Loader";
     rev = "sdk-${version}";
-    sha256 = "0w9i2pliw4ccmjyfzff4i2f3hxwsfd54jg7ahv2v634qmx59bsbi";
+    sha256 = "0b0gn4p1nz4m1lmfm8hf8xyw2fkk6c7iq6c9lg57i8z0l8crwa57";
   };
 
-  nativeBuildInputs = [ pkg-config cmake ];
-  buildInputs = [ python3 xlibsWrapper libxcb libXrandr libXext wayland ];
```

Comment:
I replaced `xlibsWrapper` with `libX11`. It includes additional unnecessary dependencies. See: https://github.com/MetaDark/nixpkgs/blob/b392cbe6fca6e3760cba470d0865c2e6e30f824a/pkgs/top-level/all-packages.nix#L18133-L18139

Reply:
lgtm!